haas/test_common.py

```diff
@@ -127,6 +127,7 @@ def newDB():
 def releaseDB():
     """Do we need to do anything here to release resources?"""
     with app.app_context():
+        db.reflect()
```

Comment:
I don't see a comment...?

What it does is try to infer the database schema by actually looking at the database, as opposed to assuming it reflects the models declared in the code. I don't want to just shove this in there without at least understanding why.